ffs/api/api.go

```diff
@@ -96,9 +106,43 @@ func (i *API) ID() ffs.APIID {
 	return i.cfg.ID
 }
 
-// WalletAddr returns the Lotus wallet address.
-func (i *API) WalletAddr() string {
-	return i.cfg.WalletAddr
+// WalletAddr returns the wallet addresses.
+func (i *API) WalletAddrs() map[string]AddrInfo {
+	return i.cfg.Addrs
+}
+
+func (i *API) GetDefaultCfg() ffs.DefaultCidConfig {
+	return i.cfg.DefaultCidConfig
+}
```

Comment:
Added this because it will be convenient to just fetch the default config without worrying about prepping for a cid.